sudo/build.rs

```diff
@@ -222,6 +222,11 @@ fn main() -> io::Result<()> {
     Command::new("powershell")
         .arg("-NoProfile")
         .arg("-c")
+        .arg("Set-ExecutionPolicy")
+        .arg("Bypass")
+        .arg("-Scope")
+        .arg("Process")
+        .arg("-Force;")
```

Comment:
FWIW, we should prefer `powershell -NoProfile -ExecutionPolicy Bypass -c foo-bar-baz` instead of trying to `set-executionpolicy` as an inline script 🙂